backend/app/routers/paybill.py

```
@@ -1,12 +1,10 @@
-
-
 from fastapi import FastAPI, Depends, HTTPException
 from sqlalchemy.orm import Session
-import user_management.schemas as schemas, user_management.models as models
-from user_management.database import get_db
-from user_management.models import User, Order, OrderItem, Bill, Product
-from user_management.schemas import PayBillRequest
-from user_management.utils import get_current_user
+import app.schemas as schemas, app.models as models
```

Ruff (0.8.2)

Multiple imports on one line

Split imports

(E401)